.github/workflows/nix.yml

```diff
@@ -0,0 +1,14 @@
+name: "Test build nix"
```

Comment:
This workflow simply runs `nix build` so if there comes a error it can be caught